### PR TITLE
DOCS-1824: Update readme

### PR DESCRIPTION
**Pull request by andf-viam**
Updating to match new [standard Readme template](https://docs.viam.com/registry/create/#optional-create-a-readme).
- Adding instructions from linked RGB-D Overlay Module Usage Instructions doc

### CODE REVIEW

**Comment by andf-viam** [comment]
Hi @hexbabe , quick update to this Readme to conform to the standard module readme template, and to add your instructions from DOC-1824. Let me know if you'd like to see any changes -- thanks!

**Review by npentrel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,46 +1,53 @@
-# RGB Depth Overlay C++ Module
+# RGB Depth Overlay modular component
 
-The RGB-D overlay module is designed to process color and depth outputs from a camera module, calling `get_images`. It uses ImageMagick to overlay the two images semi-transparently to directly compare the outputs and check if they are aligned.
+This module implements the [Viam camera API](https://docs.viam.com/build/program/apis/#camera) in a `rgb-d-overlay` model.
+With this model, you can process color and depth outputs from multiple camera modules, in order to ensure that the FOV, resolution, aspect ratio, and time synchronization of the cameras are aligned.
+This model uses ImageMagick to overlay two images semi-transparently to directly compare two camera outputs.
```
Comment (hexbabe) [inline README.md]: model -> module?

File: README.md [hunk 2]
```diff
@@ -1,46 +1,53 @@
...
+> [!NOTE]
+> For more information, see [Configure a Machine](https://docs.viam.com/manage/configuration/).
+
+### Configure your image stream camera
+
+On the camera component that supports outputting simultaneous depth and color image streams, such as the [Intel Realsense](https://app.viam.com/module/viam/realsense) or the [Luxonis OAK-D](https://app.viam.com/module/viam/oak-d), configure the `sensors` attribute to support both types of camera streams supported:
+
+Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com/).
+Click on the **Components** subtab and select the configuration pane for your image stream camera, _not_ the `rgb-d-overlay` camera.
+Under **Attributes**, add the following `sensor` configuration to your image stream camera configuration:
```
Comment (hexbabe) [inline README.md]: `sensor` -> `sensors`

**Review by hexbabe — APPROVED** [review]
A few typos? LGTM otherwise though. Thank you so much for doing this!

File: README.md (hunk 1 above)
Comment (andf-viam) [inline README.md]:
```suggestion
This module uses ImageMagick to overlay two images semi-transparently to directly compare two camera outputs.
```

File: README.md (hunk 2 above)
Comment (andf-viam) [inline README.md]:
```suggestion
Under **Attributes**, add the following `sensors` configuration to your image stream camera configuration:
```

File: README.md [hunk 3]
```diff
@@ -57,3 +64,55 @@ Example configuration snippet:
...
+   cmake ..
+   make
+   ```
+
+The outputted binary `rgb-d-overlay` will be written to the `build` directory.
+
+### Build on Linux
+
+1. [Install Docker](https://docs.docker.com/engine/install/).
+1. Clone this repository.
```
Comment (andf-viam) [inline README.md]:
```suggestion
1. Clone this repository:
```

**Pull request merged**